templates/contracts/rust/deploy.sh

```diff
@@ -10,4 +10,4 @@ fi
 echo ">> Deploying contract"
 
 # https://docs.near.org/tools/near-cli#near-dev-deploy
-near dev-deploy --wasmFile ./target/wasm32-unknown-unknown/release/hello_near.wasm
+near dev-deploy --wasmFile ./target/wasm32-unknown-unknown/release/*.wasm
```

Comment:
*.wasm is too generic, either we make it take the name of the project, or we force it to be hello-near.wasm. The last option might make more sense.